crates/matrix-sdk-crypto/CHANGELOG.md

```diff
@@ -20,6 +20,11 @@ All notable changes to this project will be documented in this file.
 
 ### Features
 
+- [**breaking**] `OlmMachine.receive_sync_changes` returns now a list of `ProcessedToDeviceEvent` 
+  instead of a list of `Raw<AnyToDeviceEvent>`. With variants like `Decrypted`|`UnableToDecrypt`|`PlainText`|`NotProcessed`.
+  This allows for example to make the difference between an event sent in clear and an event successfully decrypted.
+  For quick compatibility a helper `ProcessedToDeviceEvent::to_raw` allows to map back to the previous behaviour.
+
```

Comment:
@BillCarsonFr @poljar, a reminder to please make sure that changelog entries go in the right section of the changelog.

This change was *not* part of matrix-sdk-crypto v0.11.0.

(Also, please try to link to the PR in changelog entries)

Fixed in https://github.com/matrix-org/matrix-rust-sdk/pull/5136

Reply:
Ah right. I should more thoroughly check changelog entries in PRs. Sorry.